kahuna/public/js/directives/gr-image-fade-on-load.js

```diff
@@ -7,41 +7,52 @@ imageFade.directive('grImageFadeOnLoad',
                      function ($q, $timeout) {
 
     // TODO: customise duration, transition
-    const animationThreshold = 50; // ms
-    const animationDuration = 200; // ms
+                         const animationDuration = 200; // ms
+                         const revealAfter = 1500; // ms
```

Comment:
[nit] think this indentation is accidental

Reply:
yeah, the joys of prettier